modules/training/ajax/getTabContent.php

```diff
@@ -139,4 +154,22 @@ function createTabHTML($contentType, $title, $tabVariables, $type)
 
     return $html;
 }
+
+/**
+ * Curl helper function.
+ *
+ * @param string $url url to get
+ *
+ * @return string
+ */
+function Curl_get($url)
+{
+    $curl = curl_init($url);
```

Comment:
Curl is a PECL extension that isn't in the base PHP installation and needs to be compiled into it. I think this is a pretty heavy requirement to add just for vimeo videos.